examples/with-google-analytics/pages/_app.js

```diff
@@ -14,7 +15,30 @@ const App = ({ Component, pageProps }) => {
     }
   }, [router.events])
 
```

Comment:
Is this part still necessary?

Reply:
This tracks page views when the route changes